x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DateTimeParseProcessorTests.java

```diff
@@ -130,5 +188,18 @@ public void testParsing() {
                 .asProcessor()
                 .process(null)
         );
+        // TimeParse
+        assertEquals(
+                time(10, 20, 30, 123000000, zoneId),
+                new TimeParse(Source.EMPTY, l("10:20:30.123"), l("HH:mm:ss.SSS"), zoneId).makePipe()
+                        .asProcessor()
+                        .process(null)
+        );
+        assertEquals(
+                time(10, 20, 30, 123456789, ZoneOffset.of("+05:30"), zoneId),
+                new TimeParse(Source.EMPTY, l("10:20:30.123456789 +05:30"), l("HH:mm:ss.SSSSSSSSS zz"), zoneId).makePipe()
+                        .asProcessor()
+                        .process(null)
```

Comment:
These lines do not align with the previous tests in the same method. Please, use the same indentation rules.